### ISSUE TITLE
Carthage Support

### ISSUE THREAD

**Issue by StevenMasini** [Alecrim/AlecrimCoreData#105]
Hi everyone.
It seems there is no support for Carthage yet.

carthage update --platform iOS
*** Cloning AlecrimCoreData
*** Checking out AlecrimCoreData at "4.0.3"
*** xcodebuild output can be found in /var/folders/jt/skm6ggy95l18tbmfvgwkh7f80000gn/T/carthage-xcodebuild.GZ1FUj.log
*** Building scheme "Alamofire iOS" in Alamofire.xcworkspace
**Dependency "AlecrimCoreData" has no shared framework schemes**

**If you believe this to be an error, please file an issue with the maintainers at https://github.com/Alecrim/AlecrimCoreData/issues/new**

Can we do something about it ?
Thanks

**Comment by Dschee** [comment]
I would also like to have Carthage support and just found this: https://github.com/Alecrim/AlecrimCoreData/pull/47

I hope there is a misunderstanding of Carthage in that answer of @vmartinelli and Carthage can still be supported. Otherwise this project will be useless for many people who might have considered it otherwise, just as I do right now.

**Comment by vmartinelli** [comment]
Hello.

@StevenMasini  I do not see any reason to duplicate this issue asking for the same thing. A “+1” here and I’ll understand. ;)

@Dschee I don’t think the entire framework is useless because it does not support Carthage. ;)

OK. I really would like **ACD** having support to CocoaPods, Carthage and Swift Package Manager while keeping only one target that can be build for OS X, iOS, watchOS, tvOS and toasterOS. Any help regarding this will be appreciated.

Thank you.

**Comment by vmartinelli** [comment]
I think this a bug related to Carthage and not to **ACD**. But this is, of course, my opinion. As I did not have time to follow developments of Carthage, does anyone have any news about Carthage supporting “universal frameworks”?

**Comment by ikesyo** [comment]
Carthage did support “universal frameworks” 4 months ago: https://github.com/Carthage/Carthage/releases/tag/0.9

**Comment by vmartinelli** [comment]
Well... In this case the error trying to include **ACD** as a dependency persist?

**Comment by ikesyo** [comment]
I could successfully build this library. That should be a version issue.

@StevenMasini Are you really using the latest version [0.11](https://github.com/Carthage/Carthage/releases/tag/0.11)?

**Comment by vmartinelli** [comment]
Thank you @ikesyo. :) I think this issue can be closed too. @StevenMasini @Dschee Any progress?

**Comment by Dschee** [comment]
Hi guys. I don't remember what the issue was when I posted my comment (but I guess it was similar to what @StevenMasini wrote) but I just tried again and it is now working without any change. Can't explain but I might have done it incorrectly the last time. In any case I also want to clear my previous state: Of course this project isn't useless without Carthage support, it just would be useless **for me** without it. ;)

From my perspective, this can be closed now, too.
